metadata-integration/java/spark-lineage-legacy/README.md

```diff
@@ -96,9 +96,10 @@ The Spark agent can be configured using Databricks Cluster [Spark configuration]
 - Open Databricks Cluster configuration page. Click the **Advanced Options** toggle. Click the **Spark** tab. Add below configurations under `Spark Config`.
 
     ```text
-    spark.extraListeners                datahub.spark.DatahubSparkListener
-    spark.datahub.rest.server           http://localhost:8080
-    spark.datahub.databricks.cluster    cluster-name<any preferred cluster identifier>
+    spark.extraListeners                    datahub.spark.DatahubSparkListener
+    spark.datahub.rest.server               http://localhost:8080
+    spark.datahub.stage_metadata_coalescing true
+    spark.datahub.databricks.cluster        cluster-name<any preferred cluster identifier>
```

Comment:
this is for the legacy spark plugin - did you mean to update docs elsewhere?

Reply:
updated to update the main spark documentation instead - this is where the update was intended to be added but it was added in the wrong file.